storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1539,8 +1539,7 @@ class Rdb_transaction
                                                 false /* just statement */);
       return HA_ERR_LOCK_DEADLOCK;
     }
-
-    if (s.IsBusy())
+    else if (s.IsBusy())
```

Comment:
Right, but if isDeadlock() returns true then we would have returned already and never gotten to the `if (s.IsBusy())`.  I'm fine if you want the `else` - I'm just trying to understand why so if there is a good reason I can add it in the future.